jenkins/ci.suse.de/templates/cloud-crowbar-pipeline-template.yaml

```diff
@@ -170,7 +170,7 @@
           multi-select-delimiter: ','
           # NOTE: keeping designate and octavia disabled by default, until their
           # implementation is complete
-          default-value: '{enabled_services|database,rabbitmq,keystone,swift,monasca,glance,cinder,neutron,nova,horizon,ceilometer,heat,manila,trove,barbican,magnum,sahara,aodh,tempest}'
+          default-value: '{enabled_services|designate,database,rabbitmq,keystone,swift,monasca,glance,cinder,neutron,nova,horizon,ceilometer,heat,manila,trove,barbican,magnum,sahara,aodh,tempest}'
```

Comment:
Would be nice if you could keep the same ordering as the `value` parameter.

Reply:
fixed